salesforce_api/gsheet-new-row-to-sfdc-new-contact/sample.bal

```diff
@@ -0,0 +1,78 @@
+import ballerina/log;
+import ballerinax/googleapis.sheets;
+import ballerinax/salesforce as sfdc;
+
+// Types
+type GSheetOAuth2Config record {
+    string clientId;
+    string clientSecret;
+    string refreshToken;
+    string refreshUrl = "https://www.googleapis.com/oauth2/v3/token";
+};
+
+type SalesforceOAuth2Config record {
+    string clientId;
+    string clientSecret;
+    string refreshToken;
+    string refreshUrl = "https://login.salesforce.com/services/oauth2/token";
+};
+
+// Constants
```

Comment:
Ditto.

Reply:
Updated in [86c29e1](https://github.com/ballerina-guides/integration-samples/pull/31/commits/86c29e1236dcc64dd4c790e62e2633241a8a8307)